mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
I believe the results may be truncated if you use `FULL` here, while with `NO_TRACKS` you remove a lot of data from the original request, making sure you get all albums. I might be wrong, though.